packages/adblocker-webextension/src/index.ts

```diff
@@ -128,19 +132,74 @@ export function updateResponseHeadersWithCSP(
   return { responseHeaders };
 }
 
+const READABLE_MIME_TYPES = new Set([
```

Comment:
how about calling it `HTML_FILTERABLE_MIME_TYPES`?

Reply:
Updated in [`bead6e9` (#14)](https://github.com/seia-soto/adblocker/pull/14/commits/bead6e965e01aab9d95de53b1f0c5f2efbdc3385)